src/framework/uicomponents/qml/MuseScore/UiComponents/GradientTabButton.qml

```diff
@@ -38,6 +38,7 @@ RadioDelegate {
     property var normalStateFont: ui.theme.tabFont
     property var selectedStateFont: ui.theme.tabBoldFont
 
+    property bool isDoToggleOnNavigationActive: true
```

Comment:
maybe "isTogglingEnabled/Available" would sound better?

Reply:
It is important here that when "navigation is activated" (focus transition), that is, the toggled signal occurs when the button is clicked, and in this case, it can be sent or not sent when the focus is moved

Reply:
It doesn't matter anymore, in the next PR I will remove this functionality